frontend/app/src/KeycloakDevTools.vue

```diff
@@ -0,0 +1,142 @@
+<template>
+  <div>
+    <div class="col1">
+      <h2>User Account</h2>
+      <v-btn size="small" @click="loadProfile">Get Profile</v-btn>
+      <v-btn size="small" @click="updateProfile">Update profile</v-btn>
+    </div>
+    <div class="col1">
+      <h2>Token Information</h2>
+      <v-btn size="small" @click="loadUserInfo">Get User Info</v-btn>
+      <v-btn size="small" @click="showToken">Show Token</v-btn>
+      <v-btn size="small" @click="showRefreshToken">Show Refresh Token</v-btn>
+      <v-btn size="small" @click="showIdToken">Show ID Token</v-btn>
+      <v-btn size="small" @click="showExpires">Show Expires</v-btn>
+      <v-btn size="small" @click="showDetails">Show Details</v-btn>
+    </div>
+    <div class="col4">
+      <label style="font-weight: 600">Result</label>
+      <pre
+        id="output"
+        style="
+          background-color: #ddd;
+          border: 1px solid #ccc;
+          padding: 10px;
+          word-wrap: break-word;
+          white-space: pre-wrap;
+          margin-bottom: 20px;
+        "
+        >{{ result }}</pre
+      >
+    </div>
+  </div>
+</template>
+<script>
+export default {
+  name: 'KeycloakDevTools',
+  data: function () {
+    return {
+      result: '',
+    };
+  },
+  methods: {
+    loadUserInfo: function () {
+      var vm = this;
+      this.$keycloak
+        .loadUserInfo()
+        .success(function (responseData) {
+          vm.result = JSON.stringify(responseData, null, '  ');
+        })
+        .error(function () {
+          vm.result = 'Failed to load user info';
+        });
+    },
+    loadProfile: function () {
+      var vm = this;
+      this.$keycloak
+        .loadUserProfile()
+        .success(function (profile) {
+          vm.result = JSON.stringify(profile, null, '  ');
+        })
+        .error(function () {
+          vm.result = 'Failed to load user info';
+        });
+    },
+    showToken: function () {
+      this.result = this.$keycloak.tokenParsed;
+    },
+    showRefreshToken: function () {
+      this.result = this.$keycloak.refreshTokenParsed;
+    },
+    showIdToken: function () {
+      this.result = this.$keycloak.idTokenParsed;
+    },
+    showDetails: function () {
+      this.result = this.$keycloak;
+    },
+    showExpires: function () {
+      if (!this.$keycloak.tokenParsed) {
+        this.result = 'Not authenticated';
+        return;
+      }
+      var o =
+        'Token Expires:\t\t' +
```

Comment:
<img width="1326" alt="Screenshot 2024-03-04 at 4 10 38 PM" src="https://github.com/bcgov/moh-phlat/assets/107434397/07f68636-064d-4812-ba1b-a731e66c27d7">
On local environemnt I am using this component for testing. It would never be rendered on prod and would be removed completely once the KC setup on dev and tested. Hope thats fine with you @Ashwani-cgi